ddl/db_test.go

```diff
@@ -2672,3 +2672,133 @@ func (s *testDBSuite) TestAddPartitionTooManyPartitions(c *C) {
 	);`
 	s.testErrorCode(c, sql3, tmysql.ErrTooManyPartitions)
 }
+
+func checkPartitionDelRangeDone(c *C, s *testDBSuite, partitionPrefix kv.Key) bool {
+	hasOldPartitionData := true
+	for i := 0; i < waitForCleanDataRound; i++ {
```

Comment:
How can we make sure that the GC is done after waitForCleanDataRound * waitForCleanDataInterval time?

Reply:
GC interval is `gcWorkerTickInterval  = time.Minute`.

Reply:
@winkyao 
1.  `waitForCleanDataRound` indicates how many times should we check data is cleaned or not.
2.  `waitForCleanDataInterval` is a min duration between 2 check for data clean.

Reply:
GC run interval is `gcDefaultRunInterval = 10 * minutes`